app/controllers/cloud_network_controller.rb

```diff
@@ -282,11 +279,20 @@ def process_cloud_networks(networks, operation)
           :userid       => session[:userid]
         }
         AuditEvent.success(audit)
-        network.delete_network
+        begin
+          network.delete_network
+          deleted_networks += 1
+        rescue NotImplementedError
```

Comment:
@gildub At this point it cannot go that way. My PR as a blocker will be backported before yours so I cannot count on them 😞